shop/rest/serializers.py

```diff
@@ -298,11 +298,22 @@ def get_cart(self, instance):
 
 
 class CustomerSerializer(serializers.ModelSerializer):
-    salutation = serializers.CharField(source='get_salutation_display')
+    number = serializers.CharField(source='get_number')
 
     class Meta:
         model = CustomerModel
-        fields = ('salutation', 'first_name', 'last_name', 'email', 'extra',)
+        exclude = ('user', 'recognized', 'last_access')
+
+    def get_field_names(self, declared_fields, info):
+        fields = set(super(CustomerSerializer, self).get_field_names(declared_fields, info))
+        fields.update(['first_name', 'last_name', 'email'])
+        return list(fields)
+
+    def build_standard_field(self, field_name, model_field):
+        field_class, field_kwargs = super(CustomerSerializer, self).build_standard_field(field_name, model_field)
+        if issubclass(field_class, serializers.ChoiceField):
```

Comment:
I'm wondering, why DRF doesn't do this itself. Django provides a ``get_FOO_display()`` method for every field with choices. And DRF converts these fields into a ``serializers.ChoiceField`` without taking advantage of that method. Therefore IMHO that's a safe use-case.